members.php

```diff
@@ -200,22 +200,27 @@ function members() {
 						echo "(Last active $days days ago)";
 					}
 				}
-				
+
 			echo "</td></tr>
 			<tr><td style='width:50%;text-align:center;'>
 			<span style='font-size:1.3em;font-family:\"tempus sans itc\";font-weight:bold;'>" . $viewUser->user_name . "</span><br />
 			<img src='{$viewUser->avatar_link}' style='margin-top:5px;max-width:$avatar_size;max-height:$avatar_size;' /><br />
 			</td>";
-			
-			
+
+
 			echo "<td style='width:50%;'>
 			<label style='width:6em;'>Level:</label> 	$viewUser->level<br />
 			<label style='width:6em;'>Exp:</label> 	$viewUser->exp<br />
 			<label style='width:6em;'>Rank:</label> 	" . $ranks[$viewUser->rank] . "<br />" .
-			"<br />
-			<label style='width:6em;'>Gender:</label> 	$viewUser->gender<br />
+			"<br />";
+			if($viewUser->gender == "None") {

```

Comment:
```suggestion
			if($viewUser->gender == User::GENDER_NONE) {

```
Let's add constants for all the gender options on the User class

Reply:
done